### ISSUE TITLE
ImportError: cannot import name 'NegativeSampling'

### ISSUE THREAD

**Issue by tpoerschke** [massquantity/LibRecommender#32]
Hi :)

I'm getting the following traceback when importing the SVDpp algorithm:

## Error

Import statement:
```
from libreco.algorithms import SVDpp
```

Traceback:
```
Traceback (most recent call last):
  File "experiment.py", line 1, in <module>
    from libreco.algorithms import SVDpp
  File "/Users/timpo/.conda/envs/libreco/lib/python3.6/site-packages/libreco/algorithms/__init__.py", line 3, in <module>
    from .svd import SVD
  File "/Users/timpo/.conda/envs/libreco/lib/python3.6/site-packages/libreco/algorithms/svd.py", line 20, in <module>
    from ..utils.sampling import NegativeSampling
  File "/Users/timpo/.conda/envs/libreco/lib/python3.6/site-packages/libreco/utils/sampling.py", line 5, in <module>
    from ..data.sequence import user_interacted_seq
  File "/Users/timpo/.conda/envs/libreco/lib/python3.6/site-packages/libreco/data/__init__.py", line 1, in <module>
    from .dataset import DatasetPure, DatasetFeat
  File "/Users/timpo/.conda/envs/libreco/lib/python3.6/site-packages/libreco/data/dataset.py", line 6, in <module>
    from .transformed import TransformedSet
  File "/Users/timpo/.conda/envs/libreco/lib/python3.6/site-packages/libreco/data/transformed.py", line 4, in <module>
    from ..utils.sampling import NegativeSampling
ImportError: cannot import name 'NegativeSampling'
```

However, if you import anything from `libreco.data` first, no error occurs:
```
from libreco.data import random_split, DatasetPure
from libreco.algorithms import SVDpp
```
These two lines will not produce any error.

System information:
Python 3.6.12 :: Anaconda, Inc.
macOS Catalina 10.15.7

---

## Expected behavior

Importing an algorithm should not raise any errors.

**Comment by massquantity** [comment]
Yes it's a bug, will be fixed in the next version. Thanks!

**Comment by massquantity** [comment]
fixed in version 0.4.0
